### DISCUSSION TITLE
Enhancement of MMRotate Visualization

### DISCUSSION THREAD

**Discussion by akmalulkhairin** [open-mmlab/mmrotate#107] [Ideas]
First of all, I would like to thank MMRotate team for their stellar work on this library, I have been looking forward to using it since it was first announced. 

On that note, I would like to know if there is any plan to enhance the visualization part of the rotated detection such that it is on par with MMDet results visualization. e.g. using matplotlib to draw polygons

Thank you

**Comment by yangxue0827** [comment]
Thanks for your suggestion, and we welcome your PR too.
